tests/foreman/ui/test_role.py

```diff
@@ -298,11 +298,11 @@ def test_positive_create_filter_admin_user_with_locs(self):
             )
             self.assertIsNotNone(
                 self.role.wait_until_element(common_locators['alert.success']))
-            assigned_permissions = self.role.filters_get_permissions(
+            assigned_perms = self.role.filters_get_permissions_by_resource(
```

Comment:
seems this test was not run:  changed function take 3 args but receive only 2, and do not receive list as param